vcpkg-python-scripts/vcpkg_python_functions.cmake

```
@@ -94,4 +94,22 @@ function(vcpkg_python_build_and_install_wheel)
 
   vcpkg_python_build_wheel(${opts} SOURCE_PATH "${arg_SOURCE_PATH}" OUTPUT_WHEEL WHEEL OPTIONS ${arg_OPTIONS})
   vcpkg_python_install_wheel(WHEEL "${WHEEL}")
-endfunction()
\ No newline at end of file
+endfunction()
+
+function(vcpkg_python_test_import)
+    cmake_parse_arguments(
+        PARSE_ARGV 0
+        "arg"
+        ""
+        "MODULE"
+        ""
+    )
+
+  message(STATUS "Testing package!")
+  set(ENV{PYTHONPATH} "${CURRENT_PACKAGES_DIR}/lib/python${PYTHON3_VERSION_MAJOR}.${PYTHON3_VERSION_MINOR}/site-packages")
```

I have a variable for that since it is platform dependent:
https://github.com/Neumann-A/my-vcpkg-ports/blob/master/python3/vcpkg-port-config.cmake#L11
```suggestion
  set(ENV{PYTHONPATH} "${PYTHON3_SITE}")
```